students/junyoung-lee/user/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+class User(models.Model):
+    name        = models.CharField(max_length=50, unique=True)
+    nickname    = models.CharField(max_length=50)
+    email       = models.EmailField(unique=True)
+    phonenumber = models.CharField(max_length=13, unique=True)
```

Comment:
두 단어로 이루어진 경우 _ 를 사용해주세요!
```suggestion
    phone_number = models.CharField(max_length=13, unique=True)
```

Reply:
수정했습니다!